src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/RequestSql/form.html.twig

```diff
@@ -38,15 +38,29 @@
           </h3>
           <div class="card-block p-3">
             <div class="form-group">
+              {% set errorClass = requestSqlForm.request_sql.name.vars.errors is not empty ? 'is-invalid' : '' %}
+
               <label for="{{ requestSqlForm.request_sql.name.vars.id }}" class="form-control-label">{{ 'SQL query name'|trans }}</label>
-              {{ form_errors(requestSqlForm.request_sql.name) }}
-              {{ form_widget(requestSqlForm.request_sql.name) }}
+              {{ form_widget(requestSqlForm.request_sql.name, {'attr': {'class': errorClass}}) }}
+
+              {% for error in requestSqlForm.request_sql.name.vars.errors %}
```

Comment:
ok, although I like to finally see in-form errors maybe this should be managed in the form_errors helper in some way
otherwise we are still gonna have to make more and more complicated form templates, whereas we should be able to simply use `form_form(form)`